tests/test_autolysis.py

```diff
@@ -34,19 +34,23 @@ def setUpModule():
             logging.info('Downloading %s', dataset['path'])
             urlretrieve(dataset['url'], path)
 
+
 class TestTypes(unittest.TestCase):
     'Test autolysis.types'
     longMessage = True
 
     def check_type(self, result, expected, msg):
         'result = expected, but order does not matter. Both are dict of lists'
-        self.assertEqual(set(result.keys()), set(expected.keys()), '%s keys' % msg)
+        self.assertEqual(set(result.keys()),
+                         set(expected.keys()), '%s keys' % msg)
         for key in expected:
-            self.assertEqual(set(result[key]), set(expected[key]), '%s - %s' % (msg, key))
+            self.assertEqual(set(result[key]),
+                             set(expected[key]), '%s - %s' % (msg, key))
 
```

Comment:
In my opinion, this would be more readable if it were like
```python
self.assertEqual(set(result[key]), set(expected[key]),
                          '%s - %s' % (msg, key))
```
if line is too long.